contracts/Liquidation.sol

```diff
@@ -45,6 +47,9 @@ contract Liquidation is ILiquidation, Ownable {
         uint256 timestamp
     );
     event InvalidClaimOrder(uint256 indexed receiptId);
+    event ReleaseTimeUpdated(address indexed owner, uint256 releaseTime);
+    event MinimumLeftoverGasCostMultiplierUpdated(address indexed owner, uint256 minimumLeftoverGasCostMultiplier);
+    event MaxSlippageUpdated(address indexed owner, uint256 maxSlippage);
```

Comment:
Is there any reason to have the addresses in these events be indexed?

Reply:
Left as indexed to allow for users to sort through events by the owner. However, since owner changes are probably not that frequent I can remove.

Reply:
I guess it's more so that it's kind of independent of the owner (i.e. who the owner is doesn't really matter anyways). I think it's not worth having the owner param in the events anyways